.stickler.yml

```diff
@@ -0,0 +1,7 @@
+linters:
+  stylelint:
+    config: 'stylelint.config.js'
+
+files:
+  ignore:
+    - '*.scss'
```

Comment:
Why are you ignoring SCSS if I may ask?

Reply:
Hi Oladele,
I'm ignoring scss files from stickler validation, because stickler is rendering them and throwing errors. The problem is that stickler recognizes all files ending with css therefore is recognizing scss files as css files.

Reply:
Ah, I see. I didn't know this, thanks for the enlightenment.